Errors when setting custom CHE_CONTAINER value with multiuser mode
kind/bug status/open-for-dev team/platform 
Description I have 2 issues when trying to run che with multi user: Invalid parameter, redirect_uri when accessing to che with multiuser mode and with CHE_CONTAINER containing a - This one can be reproduced with the command docker run -it -e CHE_MULTIUSER=true -e CHE_HOST=${EXTERNAL_IP}  -e CHE_CONTAINER=che-multi -v /var/run/docker.sock:/var/run/docker.sock -v ~/.che-multiuser:/data eclipse/che start
 so accessing to EXTERNAL_IP:8080 brings us to the page from keycloak: From keycloak logs: 12:34:40,406 WARN  [org.keycloak.events] (default task-16) type=LOGIN_ERROR, realmId=che, clientId=che-public, userId=null, ipAddress=192.168.0.11, error=invalid_redirect_uri, redirect_uri=http://192.168.0.11:8081/dashboard/
 The issue seems to be related to the - contained in the container name che-multi. With CHE_CONTAINER=che-multi, it fails but with CHE_CONTAINER=chemulti, it works. I suspect it is related to the generated containers name for keycloak and postgres (che-multi, chemulti_postgres_1, chemulti_keycloack_1 insteadof che-multi, che-multi_postgres_1, che-multi_keycloak_1) Also if we only set CHE_PORT to 8081, it fails as well as the container name becomes che-8081 Can not start a workspace So if I set any value that is not che as a container name, I can not start a workspace: Good to know: I also had another error message about che not being able to find network che-multi_che..... but can not get this error back again. OS and version: RHEL 7 (CSB)